mlflow/models/python_api.py

```diff
@@ -235,19 +233,26 @@ def predict(
         pyfunc_backend_env_root_config = {"create_env_root_dir": True}
 
     def _predict(_input_path: str):
-        return get_flavor_backend(
-            model_uri,
-            env_manager=env_manager,
-            install_mlflow=install_mlflow,
-            **pyfunc_backend_env_root_config,
-        ).predict(
-            model_uri=model_uri,
-            input_path=_input_path,
-            output_path=output_path,
-            content_type=content_type,
-            pip_requirements_override=pip_requirements_override,
-            extra_envs=extra_envs,
-        )
+        try:
+            return get_flavor_backend(
+                model_uri,
+                env_manager=env_manager,
+                install_mlflow=install_mlflow,
+                **pyfunc_backend_env_root_config,
+            ).predict(
+                model_uri=model_uri,
+                input_path=_input_path,
+                output_path=output_path,
+                content_type=content_type,
+                pip_requirements_override=pip_requirements_override,
+                extra_envs=extra_envs,
+            )
+        except Exception as e:
+            if is_in_databricks_runtime() and "Permission denied" in str(e):
+                raise MlflowException(
+                    "Try reattach or restart the cluster to resolve the permission issue."
```

Comment:
"The virtual environment cannot be retrieved. To resolve this issue, either detach your notebook from your running environment and reattach, or restart your compute resource to build a new environment."

Users might wonder why this is happening...